release/long_running_tests/workloads/serve.py

```diff
@@ -9,9 +9,9 @@
 from ray.cluster_utils import Cluster
 
 num_redis_shards = 1
-redis_max_memory = 10**8
-object_store_memory = 10**8
-num_nodes = 5
+redis_max_memory = 10 ** 8
+object_store_memory = 10 ** 8
+num_nodes = 1
```

Comment:
serve is having some trouble with multiple clusters on the same node https://github.com/ray-project/ray/issues/12779